src/components/admin/security/hooks/__tests__/useLoginAttempts.debug.test.ts

```
@@ -0,0 +1,76 @@
+import { renderHook } from '@testing-library/react-hooks';
```

ESLint

[error] Definition for rule '@typescript-eslint/ban-types' was not found.

(@typescript-eslint/ban-types)